deployment/terraform/info.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Info displays information about the current load-test deployment.
+func (t *Terraform) Info() error {
+	output, err := t.getOutput()
+	if err != nil {
+		return err
+	}
+
+	t.displayInfo(output)
+
+	return nil
+}
+
+func (t *Terraform) displayInfo(output *terraformOutput) {
+	fmt.Println("==================================================")
+	fmt.Println("Deployment complete 🎉. Here is the setup information:")
+	fmt.Println("Mattermost URL: http://" + output.Proxy.Value.PublicDNS)
+	fmt.Println("Instances:")
```

Comment:
Since they are all instances maybe we can use a more detailed label like "App Servers".

Reply:
done